src/Microsoft.AspNetCore.Server.HttpSys/FeatureContext.cs

```
@@ -463,7 +464,15 @@ string IHttpRequestIdentifierFeature.TraceIdentifier
             }
         }
 
-        internal async Task OnStart()
+        bool IHttpMaxRequestBodySizeFeature.IsReadOnly => Request.HasRequestBodyStarted;
+
+        long? IHttpMaxRequestBodySizeFeature.MaxRequestBodySize
+        {
+            get => Request.MaxRequestBodySize;
+            set => Request.MaxRequestBodySize = value;
```

Yeah, I was curious if we wanted to fail fast.